Server/ChatService/services/redis_rate_limiter.py

```
@@ -17,7 +17,9 @@ class RedisRateLimiter:
     def __init__(self, redis_url: str = None):
         # Redis connection - support environment variable
         import os
-        self.redis_url = redis_url or os.getenv("REDIS_URL", "redis://localhost:6379")
+        self.redis_url = redis_url or os.getenv("REDIS_URL")
+        if not self.redis_url:
+            raise ValueError("REDIS_URL environment variable is required")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)